splinter/driver/webdriver/__init__.py

```diff
@@ -600,19 +614,93 @@ def uncheck(self, name):
         self.find_by_name(name).first.uncheck()
 
     def screenshot(self, name="", suffix=".png", full=False):
+        warnings.warn('Deprecated, use `self.capture_screenshot()` instead.')
```

Comment:
What do you think about using the form ```warnings.warn('Deprecated, use `self.capture_screenshot()` instead.', FutureWarning)``` to be more explicit about the intent of the warning? It comes from https://docs.python.org/3/library/exceptions.html#FutureWarning.

The same applies for the other usages.

Reply:
Sorry for the delayed reviews here too..!
I'm all for it 😀